.mocharc.yml

```diff
@@ -1,3 +1,5 @@
 require: '@babel/register'
-recursive: true
 timeout: '10000'
+recursive: true
+spec:
+  - 'test/**/*.test.js'
```

Comment:
Tiny improvement here. Not necessary (I was trying to understand why the tests were not running) but useful.